modules/raft/src/integrationTest/java/org/apache/ignite/raft/server/RaftCounterServerAbstractTest.java

```diff
@@ -51,10 +45,11 @@
     protected static final int PORT = 20010;
 
     /** */
-    private static final MessageSerializationRegistry SERIALIZATION_REGISTRY = new MessageSerializationRegistry()
-        .registerFactory(ScaleCubeMessage.TYPE, new ScaleCubeMessageSerializationFactory())
-        .registerFactory(HandshakeStartMessage.TYPE, new HandshakeStartMessageSerializationFactory())
-        .registerFactory(HandshakeStartResponseMessage.TYPE, new HandshakeStartResponseMessageSerializationFactory());
+    private static final MessageSerializationRegistry SERIALIZATION_REGISTRY = new MessageSerializationRegistry();
+
+    static {
+        NetworkMessagesSerializationRegistryInitializer.initialize(SERIALIZATION_REGISTRY);
```

Comment:
Should this be by default in every registry?

Reply:
It should, but I need to think of a clean implementation for that, it's not very straightforward

Reply:
done, I've extracted an interface